querybook/server/logic/metastore.py

```diff
@@ -270,6 +271,13 @@ def create_table_information(
         hive_metastore_description=hive_metastore_description,
     )
 
+    # As parameter descripton is newly added, for backward compatible,
```

Comment:
this comment is somewhat misleading, it is **optional** for metastore to provide description, there is no update needed for existing metastores if the admins decide not to

Reply:
no action needed for existing metastores, just tried to explain it why we assign it separately instead of adding it directly to constructor above. 
I'll just remove this comment.